src/store/modules/Health/EventLogStore.js

```diff
@@ -0,0 +1,91 @@
+import api from '../../api';
+
+const EventLogStore = {
+  namespaced: true,
+  state: {
+    eventLogData: null
+  },
+  getters: {
+    eventLogData(state) {
+      return state.eventLogData;
+    }
+  },
+  mutations: {
+    setEventLogData(state, eventLogData) {
+      state.eventLogData = eventLogData;
+    }
+  },
+  actions: {
+    getEventLogData({ commit }) {
+      api
+        .get('/xyz/openbmc_project/logging/enumerate')
+        .then(({ data }) => {
+          const eventLog = JSON.stringify(data.data);
+          const eventLogContent = JSON.parse(eventLog);
+          let eventLogData = [];
+          const severityFlags = {
```

Comment:
Why not just have a severity string property? that could be neither low, medium, or high ? 
Or do we even need a severity property if we have a priority property?